geograypher/cameras/cameras.py

```diff
@@ -535,9 +538,27 @@ def vis(self, plotter: pv.Plotter = None, frustum_scale: float = 0.1):
         frustum = pv.PolyData(projected_vertices[:3].T, faces)
         # Unsure exactly what's going on here, but it's required for it to be valid
         frustum.triangulate()
```

Comment:
I realized this method can be cleaned up by adding the face colors as a "scalar" attribute of the mesh as follows:
```
frustum["RGB"] = pv.pyvista_ndarray(face_colors)
```
Then you don't need to return `face_colors`. To visualize on (new) line 559 you would simply do `scalars="RGB"` to tell it to look at that mesh attribute. 

Finally, this would allow you to save the color of the mesh. To save it, you'd need to modify a couple lines up so the datatype is `np.uint8` and the values are [0, 255]. Then in your example you could do `mesh.save("/tmp/mesh.ply", texture="RGB")`.

I tested that this change would still render properly using the `.vis()` method of the camera set. I also visualized a mesh and camera set together using the `render_labels` example and that worked as well.

Reply:
Sure, I'll push to this branch in a few minutes.

Reply:
Nice